### PR TITLE
releng: Image promotion for kubernetes v1.26.3

### PR DESCRIPTION
**Pull request by palnabarun**
Image promotion for kubernetes v1.26.3
This is an automated PR generated from `kpromo`
```
kpromo pr --fork palnabarun --interactive --tag v1.26.3
```

/hold
cc: @kubernetes/release-engineering

### CODE REVIEW

**Review by xmudrii — APPROVED** [review]

**Review by saschagrunert — APPROVED** [review]

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml [hunk 1]
```diff
@@ -315,6 +315,7 @@
     "sha256:9b9987c6a07813074a69c7c9aa2a603590ea3a3097be0c3b44e2d57c54ff5c0b": ["v1.26.1-rc.0"]
     "sha256:e8fca63a0a9ae8da4a666ba6d4298b430d5775386568731693f922832ea86c11": ["v1.26.2"]
     "sha256:42b7bbe05edcc15190715eef5cb533c51c03e6c71db6c0e8db85e9fd215ce482": ["v1.26.2-rc.0"]
+    "sha256:acfc9bf377a7480867b6310a60b4aeebaff69cee5bb5885ba95d9be5bd343b08": ["v1.26.3"]
```
Comment (upodroid) [inline k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:318]: Can you rerun kpromo with v3.5.1? We moved the manifests folder from k8s.gcr.io folder to registry.k8s.io folder.

**Review by upodroid — CHANGES_REQUESTED** [review]

**Comment by BenTheElder** [comment]
Hey, you'll have to move this to registry.k8s.io/, sorry we're working on https://github.com/kubernetes/enhancements/issues/3720 ref https://github.com/kubernetes/k8s.io/pull/4934 https://github.com/kubernetes/test-infra/pull/29069 

[forcing bot to process this, will reopen immediately]
/close

**Comment by palnabarun** [comment]
@upodroid @BenTheElder -- fresh PR -> https://github.com/kubernetes/k8s.io/pull/4972

/close

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml (hunk 1 above)
Comment (palnabarun) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:318]: https://github.com/kubernetes/k8s.io/pull/4948#issuecomment-1473146353

**Pull request closed without merging**